config/environments/development.rb

```diff
@@ -31,7 +31,8 @@
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.perform_caching = false
   config.action_mailer.perform_deliveries = true
-  config.action_mailer.delivery_method = ENV['ACTION_MAILER_SMTP_DELIVERY_METHOD'].to_sym
+  config.action_mailer.delivery_method = ENV['ACTION_MAILER_SMTP_DELIVERY_METHOD'] && ENV['ACTION_MAILER_SMTP_DELIVERY_METHOD'].to_sym
+  config.action_mailer.delivery_method ||= :smtp
```

Comment:
I think we want the development default to be :file or :test - I'm reading the mailer docs right now.  Let me finish that and then either merge this or propose one of those others.  Beyond that this looks great.